src/less/components/button/index.less

```diff
@@ -137,9 +140,7 @@
   }
 }
 
-/* ================================================== *\
- * COMPONENT STYLES
-\* ================================================== */
+/// ----- COMPONENT STYLES -----
```

Comment:
Revert most of the comment conversions. Comment conversions will be handled in a separate future PR.